configs/platforms/ubuntu-16.10-amd64.rb

```diff
@@ -0,0 +1,12 @@
+platform "ubuntu-16.10-amd64" do |plat|
+  plat.servicedir "/lib/systemd/system"
+  plat.defaultdir "/etc/default"
+  plat.servicetype "systemd"
+  plat.codename "yakkety"
+
+  plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/debian/pl-build-tools-release-#{plat.get_codename}.deb"
+  plat.provision_with "export DEBIAN_FRONTEND=noninteractive; apt-get update -qq; apt-get install -qy --no-install-recommends build-essential devscripts make quilt pkg-config debhelper rsync fakeroot"
+  plat.install_build_dependencies_with "DEBIAN_FRONTEND=noninteractive; apt-get install -qy --no-install-recommends "
+  plat.vmpooler_template "ubuntu-1610-x86_64"
+  plat.output_dir File.join("deb", plat.get_codename, "PC1")
```

Comment:
From the looks of things, the output directory for my fedora builds has PC1 in it already:

```
output/
output/fedora
output/fedora/f25
output/fedora/f25/PC1
output/fedora/f25/PC1/x86_64
output/fedora/f25/PC1/x86_64/puppet-agent-1.8.2.346.g8c0eb4a-1.fedoraf25.x86_64.rpm
output/fedora/f25/PC1/i386
output/fedora/f25/PC1/i386/puppet-agent-1.8.2.345.g75b49a3-1.fedoraf25.i386.rpm
```